scripts/opensearch-test-helpers/destination_connector/test-ingest-opensearch-output.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python3
-import sys
```

Comment:
what was the deal with changes here? noticed a bug? general cleanup

Reply:
For some reason this was failing for me so I started tweaking it some to get some more information. In the end, seems like it was a transient issue in the CI but left my changes in here anyways.